docs/content/quickstart/_index.md

```diff
@@ -312,6 +312,31 @@ Currently the Operator only supports the first imagePullSecret in a service acco
 A single secret with authentication for multiple registries can achieved by 
 [creating a secret from a file](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#registry-secret-existing-credentials).
 
+## Install Plugins
+Create the `quickstart_agentconfig.yaml` with the following content:
+```yaml
+apiVersion: porter.sh/v1
+kind: AgentConfig
+metadata:
+  name: agentconfig-quickstart
+  namespace: quickstart
+spec:
+  porterRepository: ghcr.io/getporter/porter-agent
+  porterVersion: v1.0.2
+  serviceAccount: porter-agent
+  volumeSize: 64Mi
+  pullPolicy: Always
+  installationServiceAccount: installation-agent
+  plugins:
+    - name: "kubernetes"
+```
+
+Create the `AgentConfig` custom resource by running `kubectl apply -f quickstart_agentconfig.yaml`
+
+If no plugins are defined in a AgentConfig, the operator will set its default value to use the `kubernetes-plugin`
+
+🚨 WARNING: Currently, the operator can only install one plugin per AgentConfig. If more than one plugins are defined in the CRD, it will omit the rest of the defined plugins but the first one.
```

Comment:
Is this because of the limitations of calling "porter" inside the agent action and installing multiple plugins would currently require multiple porter commands? Having the ability to run multiple porter commands inside a single agent action came up before when we were working on some outputs stuff and it might be a nice addition in general to have to the agent image. It would be a simple update to https://github.com/getporter/porter/blob/main/cmd/agent/main.go to add that in and we could get that behavior in addition to potentially allowing this for other use cases. I'm also fine with leaving it as is and just waiting until porter directly supports installing multiple plugins via a single plugins install command

Reply:
Yes, you are right on point. That's definitely something that worth looking into adding. However, I didn't dig too deep into the design for making that change. I have created an issue to track this https://github.com/getporter/porter/issues/2492 
If you want to add more detailed design in the issue, please feel free to!